memory/libhugetlbfs.py

```diff
@@ -100,9 +102,33 @@ def setUp(self):
         process.run('patch -p1 < %s' % data_dir + '/' + patch, shell=True)
         build.make(self.srcdir, extra_args='BUILDTYPE=NATIVEONLY')
 
+    def fail_check(self, check):
+
+        log = self.logdir + '/stdout'
+        cmd_fail = "grep 'FAIL:' %s  |head -1| awk '{ print $%s}'" % (
+            log, check)
+        if check == 3:
+            env = "32"
+        else:
+            env = "64"
+
+        fail_check = os.popen(cmd_fail).read()
+
+        if fail_check > 0:
+
+            self.fail(fail_check + " hugpage page " + env + " bit test failed")
+
     def test(self):
         os.chdir(self.srcdir)
-        build.make(self.srcdir, extra_args='BUILDTYPE=NATIVEONLY check')
+
+        # use process.system :to log 'make command' output in stdout
+        process.system('make BUILDTYPE=NATIVEONLY check')
```

Comment:
@apahim because need to log of command '`make BUILDTYPE=NATIVEONLY check'`  in avocado stdout as that is a base file to grep the failure 

as I Mentioned in comment .

Thanks 
Praveen